sample_code/ep540/README.md

```diff
@@ -0,0 +1,13 @@
+# [I made every sentry page 300ms faster (intermediate)](https://youtu.be/FY7EQwT7QKU)
+
+Today I talk about how I made every sentry.io page ~250-300 ms faster and talk a little bit about the various http caching approaches
+
+## Interactive examples
+
+### Bash
+
+```bash
+curl -I <url>
```

Comment:
placeholder instead of actual CDN link

Reply:
the actual link would be fine I think

Reply:
or you can find any one from the sentry login page -- I realize typing one of those out would be a pain in the ass

Reply:
Basically this helps the reader to think and to replace the placeholder value. I do the same with all git hashes.

Reply:
If you want, I can replace it with actual url value.

Reply:
I think they're fairly permanent

Reply:
Well, yeah for one year or whatever is the expiration time :D But the value on the actual Sentry login page already changed.
The old one `https://s1.sentry-cdn.com/_static/dist/sentry/entrypoints-hashed/sentry-B0SOKci_bIC4DQfMeYs2G7ti9GKmSLzdPvA6bYL4UZk.css` gives me Access Denied ...
So do you want the new value?

Reply:
shrugs

Reply:
Sorry for the noisy comments/communication, but I was not sure as well.
I know that you have more important stuff to do.
Have a nice day!